src/scanners.rs

```diff
@@ -964,38 +964,48 @@ pub fn is_html_tag(tag: &str) -> bool {
 pub fn scan_html_type_7(data: &str) -> Option<usize> {
     let mut i = scan_ch(data, b'<');
     if i == 0 {
-        return None; }
-    i += scan_ch(&data[i..], b'/');
+        return None;
+    }
+
+    let close_tag_bytes = scan_ch(&data[i..], b'/');
```

Comment:
Why am I seeing this diff? It doesn't seem related to link defs. Maybe some branch confusion?

Reply:
You're absolutely right. It fixes a test unrelated to links. I will make a separate PR for this.